src/app/Models/Traits/SpatieTranslatable/HasTranslations.php

```diff
@@ -14,6 +15,11 @@ trait HasTranslations
      */
     public $locale = false;
 
+    /**
+     * @var bool
+     */
+    public $useFallbackLocale = true;
```

Comment:
1 - Can't change the name as that's the name Spatie uses, see: https://github.com/spatie/laravel-translatable/blob/2b81c88beb7ca6616808d9a1cc98a0b3d0377e1a/src/HasTranslations.php#L23

2 - Done.

3 - Because Spatie defaults to true when property is not set. And yes, we use fallback locales by default (before this PR), that's why if you open an entry in any locale that's not translated you get the translation from the fallback locale instead of the empty inputs as in this PR.

Reply:
1. ok
2. ok
3. ok - if we only use it in one place, let's do that